go/adbc/sqldriver/driver.go

```diff
@@ -70,10 +70,10 @@ func getIsolationlevel(lvl sql.IsolationLevel) adbc.OptionIsolationLevel {
 	return ""
 }
 
-func parseConnectStr(str string) (ret map[string]string, err error) {
+func ParseConnectStr(str string) (ret map[string]string, err error) {
```

Comment:
I'm not a big fan of exposing this if we don't need to, can you create a new file `driver_internals_test.go` and just not use a `_test` package so you can call and test this directly in the test file without having to expose it?